tests/test_pubsub.py

```diff
@@ -474,3 +475,31 @@ def t_run(rc):
             t.start()
     except Exception:
         print("Error: unable to start thread")
+
+
+class TestPubSubPubSubSubcommands(object):
+
+    @skip_if_server_version_lt('2.8.0')
```

Comment:
In one way this blocking is kinda unnessesary becuase this lib will never test against any redis installation that is less then 3.0 becuase that was the version cluster support was introduced. I will add to my todo to remove it elsewhere later but no need to have it here.